src/main/java/edu/ucsb/cs156/courses/services/UCSBCurriculumService.java

```diff
@@ -187,6 +189,51 @@ public String getSection(String enrollCode, String quarter) {
         log.info("json: {} contentType: {} statusCode: {}", retVal, contentType, statusCode);
         return retVal;
     }
+
+    public String getAllSections(String enrollCode, String quarter) {
```

Comment:
The only thing I don't quite get here is: why is this called `getAllSections`?  Sure if it is looking up by enrollCd, it is going to get exactly one section, and one section only, right?

What is the difference between the contract of the SECTION_ENDPOINT (from v1 of the api) and ALL_SECTIONS_ENDPOINT (from v3 of the api?)

Reply:
nvm.. I get it now. but I do think we should take from your excellent writeup in the PR description, and convert that into Javadoc comment.